exporter/chronicleexporter/marshal.go

```diff
@@ -50,7 +51,6 @@ type logMarshaler interface {
 	MarshalRawLogs(ctx context.Context, ld plog.Logs) ([]*api.BatchCreateLogsRequest, error)
 	MarshalRawLogsForHTTP(ctx context.Context, ld plog.Logs) (map[string]*api.ImportLogsRequest, error)
 }
-
 type protoMarshaler struct {
```

Comment:
Since we're using dynamic labels, the `labels` object on this struct is no longer used and should be removed.